ci/requirements-dev-3.5.yml

```diff
@@ -18,7 +18,9 @@ dependencies:
   - plumbum
   - psycopg2
   - pyarrow>=0.6.0
+  - pymapd>=0.3.2
   - pymysql
+  - pytables
```

Comment:
Is this a hard dependency of mapd or pymapd?

Reply:
I just copied from 3.6.yml to have the same environment ... should I remove that?